File: tree-hugger/lib/tests/fixtures/imports.ts
// Test fixture for import extraction

// Simple named import
import { readFile } from "fs";

// Aliased import
import { readFile as read, writeFile as write } from "fs/promises";

// Namespace import
import * as path from "path";

// Default import (not captured by current query)
// import express from "express";

// Multiple imports from same source
import { join, resolve } from "path";

// Re-export (not tested here)
// export { foo } from "module";

console.log(readFile, read, write, path, join, resolve);

File: tree-hugger/lib/tests/fixtures/imports.rs
// Test fixture for Rust import extraction

// Simple use
use std::io;

// Use with path
use std::collections::HashMap;

// Use with multiple items
use std::process::{Child, Command, Stdio};

// Aliased use
use std::io::Read as IoRead;

// Glob import (not captured by current query)
// use std::prelude::*;

// External crate
extern crate serde;

fn main() {
    let _: HashMap<String, String> = HashMap::new();
}
